### ISSUE TITLE
Features: Hacking

### ISSUE THREAD

**Issue by dmf444** [OpenModularTurretsTeam/OMTControl#7]
Basically, moving my personal mod, CombatPlus into OMTC.

- [ ] Wireless Turret Hacking - Looks for turrets within a chunk, attempt to set the hacker as a friend.
  - [ ] Config to set maximum hackable level
- [ ] Advanced Wireless Turret Hacker - Sets the hacker as the turret owner
- [ ] Wireless Defender - Opposite of the hacker, looks for a hacking terminal and disables it
- [ ] Advanced Wireless Defender - Either explode the Hacker, or the Hacking terminal... TBD

**Comment by Keridos** [comment]
Yes, I support that. Should all be configurable in the mod config though.
